services/brig/src/Brig/API.hs

```diff
@@ -1382,11 +1388,21 @@ canBeDeleted (uid ::: tid) = do
        Team.IsOnlyTeamOwner       -> throwStd noOtherOwner
        Team.IsOneOfManyTeamOwners -> pure ()
        Team.IsNotTeamOwner        -> pure ()
-       Team.NoTeamOwnersAreLeft   -> do
+       Team.NoTeamOwnersAreLeft   -> do  -- (keeping the user won't help in this case)
            Log.warn $ Log.field "user" (toByteString uid)
                     . Log.msg (Log.val "Team.NoTeamOwnersAreLeft")
     return empty
 
+isTeamOwner :: UserId ::: TeamId -> Handler Response
+isTeamOwner (uid ::: tid) = do
+    onlyOwner <- lift (Team.isOnlyTeamOwner uid tid)
```

Comment:
`Team.isOnlyTeamOwner` could use a better name. Perhaps `getOwnerState` or something better. (a function `isTeamOwner` calling `isOnlyTeamOwner` is misleading)